pkg/server/commands/connected_objects.go

```diff
@@ -424,45 +384,6 @@ func (c *ConnectedObjectsCommand) reverseExpandDirect(
 	sourceObjectType := req.sourceObjectRef.GetType()
 	sourceObjectRel := req.sourceObjectRef.GetRelation()
 
-	var tuples []*openfgapb.Tuple
-	for _, t := range req.contextualTuples {
-
-		object := t.GetObject()
-		objectType, _ := tuple.SplitObject(object)
-		if objectType != ingress.GetType() {
-			continue
-		}
-
-		if t.GetRelation() != ingress.GetRelation() {
-			continue
-		}
```

Comment:
In the other piece you removed, this bit is 

```
if t.GetRelation() != tuplesetRelation {
			continue
		}
```

is this refactor safe?

Reply:
I asked Adrian to add more tests covering this: https://github.com/openfga/openfga/pull/601